rr_common/src/sound_node/sound_node.cpp

```diff
@@ -0,0 +1,32 @@
+#include <sound_play/sound_play.h>
+#include <unistd.h>
+#include <std_msgs/Bool.h>
+#include <ros/ros.h>
+using namespace std;
+
+bool raceStarted = false;
+
+void soundCB(const std_msgs::Bool::ConstPtr &bool_msg) {
+    raceStarted = bool_msg->data;
+}
+
+int main(int argc, char **argv) {
+    ros::init(argc, argv, "sound_node");
+    ros::NodeHandle nh;
+    sound_play::SoundClient sc;
+    bool lastraceStarted = false;
+
+
+    ros::Subscriber sub = nh.subscribe("/start_detected", 1, soundCB);
+    while (ros::ok){
```

Comment:
does ros::ok work? I usually see ros::ok()

Reply:
yeah, this should definitely have parenthesis